python/practices/practice-3/src/path1/required_letter_word_counter.py

```diff
@@ -0,0 +1,58 @@
+"""
+Check and print all word that uses all the letters in a string
```

Comment:
Please update to be more detailed, what exactly `string` is, or where it was imported from?

Reply:
Done